docs/react-v9/contributing/rfcs/shared/build-system/stop-styles-transforms.md

```diff
@@ -0,0 +1,162 @@
+# RFC: Stop pre-processing styles with Griffel in `@fluentui/react-components`
+
+[@layershifter](https://github.com/layershifter)
+
+## Background
+
+We've begun noticing a pattern where apps are bundled into multiple separate bundles. For example, there's the main bundle for the app itself and a separate bundle for a first UI piece fetched from a CDN. This problem is described in the following issues:
+
+- [microsoft/griffel#453](https://github.com/microsoft/griffel/issues/453) (_check this issue for more details_)
+- [microsoft/griffel#526](https://github.com/microsoft/griffel/issues/526)
+
+_TL;DR_: Griffel relies on the order of CSS classes, which can lead to clashes when multiple bundles are involved. See the simple example below:
+
+```html
+<!-- CURRENT STATE -->
+
+<!-- main bundle -->
+<style>
+  .order0 {
+    padding: 10px;
+  }
+  .order1 {
+    padding-left: 5px;
+  }
+</style>
+
+<!-- CDN bundle -->
+<style>
+  .order0 {
+    padding: 10px;
+  }
+</style>
+
+<!-- HTML -->
+
+<!--
+  🔴 We expect that a "div" below will have "padding-left: 5px", but instead, it has "padding: 10px".
+
+     The issue appears because the CDN bundle loads after the main bundle, resulting in style overrides. This occurs because ".order0" appears in both bundles, and CSS prioritizes the order of appearance. 
+  -->
+
+<div class="order0 order1"></div>
+```
+
+The single reliable solution is to prefix styles with a unique identifier. This prevents CSS rules from colliding, for example:
+
+```html
+<!-- PROPOSAL -->
+
+<!-- main bundle -->
+<style>
+  .main-order0 {
+    padding: 10px;
+  }
+  .main-order1 {
+    padding-left: 5px;
+  }
+</style>
+
+<!-- CDN bundle -->
+<style>
+  .order0 {
+    padding: 10px;
+  }
+</style>
+
+<!-- HTML -->
+
+<!--
+  ✅ Now the "div" below will have "padding-left: 5px" as CSS rules don't clash anymore.
+  -->
+
+<div class="main-order0 main-order1"></div>
+```
+
+## Problem statement
```

Comment:
@layershifter  can you please include in this RFC the current workaround that we have proposd and its cons?

Reply:
Added "Current workaround" section.